src/vectors.jl

```diff
@@ -1,4 +1,4 @@
-const pm_Vector_suppT = Union{pm_Integer, pm_Rational}
+const pm_Vector_suppT = Union{pm_Integer, pm_Rational, Float64}
```

Comment:
maybe we should merge `pm_Vector_suppT` with `pm_Matrix_suppT` to `pm_supported_element_types` and make sure both `Matrix` and `Vector` support the same types